hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -1,3 +1,7 @@
+{-# LANGUAGE InstanceSigs #-}
+{-# LANGUAGE UndecidableInstances #-}
```

Comment:
How hard is that to do?

On Thu, Jan 7, 2016 at 6:06 PM, Moritz Kiefer <notifications@github.com>
wrote:

> In hie-base/Haskell/Ide/Engine/PluginTypes.hs
> <https://github.com/haskell/haskell-ide-engine/pull/152#discussion_r49089216>
> :
>
> > @@ -1,3 +1,7 @@
> > +{-# LANGUAGE InstanceSigs #-}
> > +{-# LANGUAGE UndecidableInstances #-}
>
> Depends on who you ask, I use it fairly often. It is required for some of
> the magic singletons does. We could disable it and write the singletons
> manually.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haskell/haskell-ide-engine/pull/152/files#r49089216>.
>